charts/prometheus-snmp-exporter/README.md

```diff
@@ -27,6 +27,22 @@ _See [configuration](#configuration) below._
 
 _See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
 
+## Upgrading Chart
```

Comment:
Sorry to be a pain, there is already an `## Upgrading Chart` section on line `#56`. Please could you move the `### From 1.x to 2.x` section you wrote to be beneath that section above the existing `### To 1.0.0` section and rename it to be `### To 2.0.0` to match the existing naming convention.

I like the `### Upgrading an existing Release to a major version` section, please could you also move that to under the existing `## Upgrading Chart` section beneath the Helm upgrade command details on line `#62`.

Thanks!

Reply:
Updated, thanks!